DogTales/src/components/physics.hpp

```diff
@@ -4,20 +4,20 @@
 
 namespace component {
 
-constexpr glm::vec2 default_friction{0.999f};
-constexpr float default_gravity{-1000.f};
-constexpr float default_mass{1.f};
+class Physics {
 
-constexpr glm::vec2 max_velocity_v{1000.f};
-constexpr bave::Seconds tick_limit_v{100ms};
+	static constexpr glm::vec2 default_friction{0.99f};
+	static constexpr float default_gravity{-1000.f};
+	static constexpr float default_mass{1.f};
 
-class Physics {
+	static constexpr glm::vec2 max_velocity_v{1000.f};
+	static constexpr bave::Seconds tick_limit_v{100ms};
```

Comment:
Nitpick: some constants use a `_v` suffix (for "value", standard library convention), some don't.

Reply:
got it!